playwright/tests/search.spec.ts

```diff
@@ -130,7 +130,8 @@ test.describe("displays specific filter options", () => {
     await expect(searchPage.collectionFilter).toBeVisible();
     await searchPage.collectionFilter.click();
     await expect(searchPage.collectionOption).toBeVisible();
-    await searchPage.collectionOption.click();
+    await searchPage.collectionOption.check();
```

Comment:
@jbdalton want to call out the shift from using `click()` to `check()` here which is preferable for radio buttons like we are using here https://www.ideabrowser.com/idea/nestprep-pre-baby-home-organization-service